client/signup/steps/jpo-summary/index.jsx

```diff
@@ -0,0 +1,213 @@
+	}
+
+	getFormattedPayload() {
+		const payload = this.props.signupProgress;
+		return {
+			siteTitle: get( payload[ 0 ], [ 'jpoSiteTitle', 'siteTitle' ], '' ),
+			siteDescription: get( payload[ 0 ], [ 'jpoSiteTitle', 'siteDescription' ], '' ),
+			businessPersonal: get( payload[ 1 ], [ 'jpoSiteType', 'businessPersonal' ], '' ),
+			genre: get( payload[ 1 ], [ 'jpoSiteType', 'genre' ], '' ),
+			businessName: get( payload[ 1 ], [ 'jpoSiteType', 'addressInfo', 'businessName' ], '' ),
+			businessAddress: get( payload[ 1 ], [ 'jpoSiteType', 'addressInfo', 'streetAddress' ], '' ),
+			businessCity: get( payload[ 1 ], [ 'jpoSiteType', 'addressInfo', 'city' ], '' ),
+			businessState: get( payload[ 1 ], [ 'jpoSiteType', 'addressInfo', 'state' ], '' ),
+			businessZipCode: get( payload[ 1 ], [ 'jpoSiteType', 'addressInfo', 'zipCode' ], '' ),
+			homepageFormat: get( payload[ 2 ], 'jpoHomepage', '' ),
+			addContactForm: get( payload[ 3 ], 'jpoContactForm', '' )
+		};
+	}
+
+	completeOnboarding() {
+		// Get the payload and original JPC url
+		const payload = this.getFormattedPayload();
+		const jetpackConnectUrl = localStorage.getItem( 'jpoConnectUrl' );
+
+		// Flag the flow as complete for use in JPC
+		localStorage.setItem( 'jpoFlowComplete', '1' );
+
+		// Store the payload in localStorage for use after Jetpack is connected
+		localStorage.setItem( 'jpoPayload', JSON.stringify( payload ) );
+
+		// Redirect to the original JPC URL
+		page.redirect( jetpackConnectUrl );
+	}
+
+	renderStepContent() {
+		const connectionToJetpackComplete = true;
+
+		const checkMark = <Gridicon
+			className="plan-features__item-checkmark"
+			size={ 18 }
+			icon="checkmark"
+			/>;
+
+		const {
+			signupProgress: {
+				0: {
+					jpoSiteTitle
+				},
+				1: {
+					jpoSiteType
+				},
+				2: {
+					jpoHomepage
+				},
+				3: {
+					jpoContactForm
+				}
+			}
+		} = this.props;
+
+		const pathPrefix = '/start/jetpack-onboarding/';
+
+		return (
+			<div className="jpo__summary-wrapper">
+				<table>
+					<tbody>
+						<tr>
+							<td>
+								<div className="jpo__summary-col-header jpo-summary__col-header">
+									{
+										translate( "Steps you've completed:" )
+									}
+								</div>
+								<ul className="jpo-summary__completed-onboarding">
+									{
+										jpoSiteTitle
+											? <li>{ checkMark } { translate( 'Site Title & Description' ) }</li>
+											: null
+									}
+									{
+										jpoSiteType
+											? <li>{ checkMark } { translate( 'Type of Site' ) }</li>
+											: null
+									}
+									{
+										jpoHomepage
+											? <li>{ checkMark } { translate( 'Type of Homepage' ) }</li>
+											: null
+									}
+									{
+										jpoContactForm
+											? <li>{ checkMark } { translate( 'Contact Us Form' ) }</li>
+											: null
+									}
+									{
+										connectionToJetpackComplete
+											? <li>{ checkMark } { translate( 'Connection to Jetpack' ) }</li>
```

Comment:
Hi! I've found a possible matching string that has already been translated **25** times:
`translate( 'Jetpack Connection' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).